src/Hosting/IntegrationTesting/src/Common/DeploymentResult.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Hosting.IntegrationTesting
+{
+    /// <summary>
+    /// Result of a deployment.
+    /// </summary>
+    public class DeploymentResult
```

Comment:
Are we omitting HttpClient here because it's not needed? https://github.com/aspnet/AspNetCore/blob/18b81bacceacfd239359e6932742590e3f5094e3/src/Hosting/Server.IntegrationTesting/src/Common/DeploymentResult.cs